src/embedWebFonts.ts

```diff
@@ -106,7 +115,7 @@ export async function getCssRules(
               .then((cssText: any) => {
                 const parsed = parseCSS(cssText)
                 parsed.forEach((rule: any) => {
-                  ;(inline as CSSStyleSheet).insertRule(
+                  (inline as CSSStyleSheet).insertRule(
```

Comment:
The pre-commit hook seems to be forcing this change. Probably best if I leave it in, but we can edit it here in Github if necessary.